skbio/io/format/tests/test_embl.py

```diff
@@ -0,0 +1,773 @@
+            obs = fh.getvalue()
+
+        with open(self.multi_fp) as fh:
+            exp = fh.read()
+
+        self.assertEqual(obs, exp)
+
+    def test_dna_to_embl(self):
+        with io.StringIO() as fh:
+            for i, (seq, md, imd, constructor) in enumerate(self.multi):
+                obj = constructor(
+                    seq, md, interval_metadata=imd, lowercase=True)
+                _dna_to_embl(obj, fh)
+
+            # read all records written
+            obs = fh.getvalue()
+
+        with open(self.multi_fp) as fh:
+            exp = fh.read()
+
+        self.assertEqual(obs, exp)
+
+    def test_protein_to_embl(self):
+        seq, md, imd, constructor = self.protein
+        obj = constructor(seq, md, interval_metadata=imd)
+
+        with io.StringIO() as fh:
+            self.assertRaisesRegex(EMBLFormatError,
+                                   "There's no protein support for EMBL "
+                                   "record",
+                                   _protein_to_embl, [obj], fh)
+
+    def test_rna_to_embl(self):
+        with io.StringIO() as fh:
+            seq, md, imd, constructor = self.single_rna
+            obj = constructor(seq, md, interval_metadata=imd, lowercase=True)
+            _rna_to_embl(obj, fh)
+            obs = fh.getvalue()
+
+        with open(self.single_rna_fp) as fh:
+            exp = fh.read()
+
+        self.assertEqual(obs, exp)
+
+
+class RoundtripTests(EMBLIOTests):
+    def test_roundtrip_generator(self):
+        with io.StringIO() as fh:
+            _generator_to_embl(_embl_to_generator(self.multi_fp), fh)
+            obs = fh.getvalue()
+
+        with open(self.multi_fp) as fh:
+            exp = fh.read()
+
+        self.assertEqual(obs, exp)
+
+    def test_roundtrip_rna(self):
+        with io.StringIO() as fh:
+            _rna_to_embl(_embl_to_rna(self.single_rna_fp), fh)
+            obs = fh.getvalue()
+
+        with open(self.single_rna_fp) as fh:
+            exp = fh.read()
+
+        self.assertEqual(obs, exp)
+
+    def test_roundtrip_dna(self):
+        with io.StringIO() as fh:
+            _dna_to_embl(_embl_to_dna(self.single_rna_fp), fh)
+            obs = fh.getvalue()
+
+        with open(self.single_rna_fp) as fh:
+            exp = fh.read()
+
+        self.assertEqual(obs, exp)
+
+    # TODO: test_roundtrip_protein
+
+    def test_roundtrip_sequence(self):
+        with io.StringIO() as fh:
+            _sequence_to_embl(_embl_to_sequence(self.single_rna_fp), fh)
+            obs = fh.getvalue()
+
+        with open(self.single_rna_fp) as fh:
+            exp = fh.read()
+
+        self.assertEqual(obs, exp)
+
+
+class Convertertest(EMBLIOTests):
+    def test_gbtoembl(self):
+        genbank = DNA.read(self.genbank_fp, format="genbank")
+
+        with io.StringIO() as fh:
+            DNA.write(genbank, format="embl", file=fh)
+
+            # EMBL can't deal with genbank version (ie M14399.1  GI:145229)
```

Comment:
In Genbank parser, GI are read since the are in [VERSION](https://github.com/bunop/scikit-bio/blob/issue-1483/skbio/io/format/tests/data/genbank_single_record#L4) record. To test it with embl, I could download a record with no GI from gb